### PR TITLE
Perf/saving container notes plus on blur

### PR DESCRIPTION
**Pull request by deonjo**
Not really a perf increase. But i tidied the code a little, and added onBlur back. I took it out because it wasn't working first. But now it works again.

Screenshot 2019-08-20 17 14 55

### CODE REVIEW

File: src/components/container/editContainerNote.jsx [hunk 1]
```diff
@@ -40,9 +40,10 @@ export class EditContainerNote extends Component {
   
     render () {
         return (
-                <TextField
+            <TextField
                 multiline margin="none" placeholder="Add Note here"
                 autoFocus={true}
+                onBlur={() => this.props.handleNoteEnter(null)}
```
Comment (crystaldou) [inline src/components/container/editContainerNote.jsx]: is there a reason why you are passing null here?

Comment (deonjo) [reply src/components/container/editContainerNote.jsx]: Addressed your comment. Thanks!

Comment (jeffchiucp) [reply src/components/container/editContainerNote.jsx]: nice.

File: src/components/container/editContainerNote.jsx [hunk 2]
```diff
@@ -43,7 +43,7 @@ export class EditContainerNote extends Component {
             <TextField
                 multiline margin="none" placeholder="Add Note here"
                 autoFocus={true}
-                onBlur={() => this.props.handleNoteEnter(null)}
```
Comment (jeffchiucp) [inline src/components/container/editContainerNote.jsx]: bump

**Pull request merged**